.github/workflows/app.yml

```diff
@@ -93,7 +93,7 @@ jobs:
       - name: Generate TuistApp
         run: mise x -- tuist generate TuistApp --no-binary-cache
       - name: Build TuistApp
-        run: mise x -- tuist xcodebuild build -scheme TuistApp -destination "generic/platform=iOS Simulator" -workspace Tuist.xcworkspace CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO ARCHS="arm64"
+        run: mise x -- tuist xcodebuild build -scheme TuistApp -destination "generic/platform=iOS Simulator" -workspace Tuist.xcworkspace
```

Comment:
What about documenting this in case someone runs into a similar thing?

Reply:
I don't know how often folks build for simulator with those extra values. And it's specific to the keychain. I think it's somewhat unlikely someone will run into this. It's a tricky balance – do we document every little thing we run into, making the docs large, or we do we end up documenting not enough?

In other words, I'm all for documenting common issues, but I'm just not convinced this will be a common issue, so I'd refrain from documenting it just yet.